custom-connectors/SkyPointCloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,840 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SkyPoint Cloud Platform",
+    "description": "SkyPoint is a customer data platform that champions privacy and compliance through connected privacy-first customer experiences. SkyPoint Connector provides a set of APIs to work with SkyPoint platform instances and entities.",
+    "version": "1.0",
+    "contact": {
+      "name": "SkyPoint Cloud Support",
+      "url": "https://skypointcloud.com/customer-support/",
+      "email": "support@skypointcloud.com"
+    }
+  },
+  "host": "gw.skypointcloud.com",
+  "basePath": "/api",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://skypointcloud.com/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://privacy.skypointcloud.com/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "CDP;Website"
```

Comment:
Unfortunately `CDP` is not a valid category for the purpose of certification. The valid categories are:

`AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website`